### ISSUE TITLE
Add PyTorch image classification

### ISSUE THREAD

**Issue by fracpete** [waikato-ufdl/ufdl-job-launcher-plugins#3]
* utilize [imagenet](https://github.com/pytorch/examples/tree/master/imagenet) code (BSD3)
* options for user:
  * network to train: 
    `alexnet | densenet121 | densenet161 | densenet169 | densenet201 | resnet101 | resnet152 |  resnet18 | resnet34 | resnet50 | squeezenet1_0 | squeezenet1_1 | vgg11 | vgg11_bn | vgg13 | vgg13_bn | vgg16 | vgg16_bn | vgg19`
  * learning rate
  * epochs
  * momentum
  * weight-decay
  * batch-size
